### ISSUE TITLE
Node module completion just completes core modules

### ISSUE THREAD

**Issue by dNitro** [maralla/completor.vim#59]
When i require built in node modules like: `path`, `http`, ... i get all possible completion items on them after hitting `.` ( `|` is cursor position ):
```javascript
var path = require('path');
path.|   // <-- Here i get all possible items as soon as i hit DOT
```
But if i require a module like: *lodash* i get nothing!
```javascript
var lodash = require('lodash');
lodash.|   // <-- Here i get nothing after hitting DOT 
           //     Also manually feed <C-x><C-u> echos 'pattern not found'
```
Even using modules just do completions **First Time** that i require them, and if i close vim and come back to edit same file i don't get completions anymore:
*foo.js*
```javascript
module.exports = {
  version: '0.1.0'
}
```
*app.js*
```javascript
var foo = require('./foo');
foo.|   // <-- Here i get *version* just first time if i directly come from foo and require it
        //     and if i close vim and come back i get: 'pattern not found' on hitting DOT
        //     or <C-x><C-u> manually.

**Comment by maralla** [comment]
This issue has been fixed by pr #74.

**Comment by dNitro** [comment]
Thanks @maralla , a fast completor plugin now much enjoyable.
